server/request_response.go

```diff
@@ -50,3 +40,18 @@ func (r *RpcRequestHandler) _writeRpcBatchResponse(res []*types.JsonRpcResponse)
 		(*r.respw).WriteHeader(http.StatusInternalServerError)
 	}
 }
+
+func rpcToHttpCode(rpcStatusCode int) int {
```

Comment:
Where did you find this mapping? Shouldn't http status code be 200 even on errors, using JSON-RPC?

Reply:
yes it works as what you mentioned..we respond with 200 ok for errors. this is used in requestResponse